components/clubs-page.jsx

```diff
@@ -116,18 +120,95 @@ var BottomCTA = React.createClass({
+          </fieldset>
+          <fieldset>
+            <label>Where does it take place?</label>
+            <input type="text" placeholder="Type in a city or a country" />
+          </fieldset>
+          <fieldset>
+            <label>Does your Club have a website?</label>
+            <input type="text" placeholder="www.myclubwebsite.com" />
+          </fieldset>
+          <fieldset>
+            <label>What do you focus your efforts on?</label>
+            <textarea rows="5" placeholder="Give us a brief decription about what your Club is about." />
+          </fieldset>
+          <input type="submit" className="btn" value="Add Your Club To The Map" />
+        </form>
+      </Modal>
+    );
+  }
+});
+
+
+var ModalLearnMore = React.createClass({
+  render: function() {
+    return(
+      <Modal
+        modalTitle="Learn More About Hive Learning Clubs"
+        onClose={this.props.onClose}>
+        <form>
+          <fieldset>
+            <label>What is your first name?</label>
+            <input type="text" placeholder="We're a friendly bunch, promised!" />
+          </fieldset>
+          <fieldset>
+            <label>Where does it take place?</label>
+            <input type="text" placeholder="Type in a city or a country" />
+          </fieldset>
+          <fieldset>
+            <label>What is your e-mail?</label>
+            <p>A member of our team will personally reach out to you.</p>
+            <input type="email" placeholder="email@example.com" />
+          </fieldset>
+          <input type="submit" className="btn" value="Find Out More" />
+        </form>
+      </Modal>
+    );
+  }
+});
+
+
 var ClubsPage = React.createClass({
   statics: {
     pageClassName: "clubs"
   },
+  getInitialState: function() {
+    return {
+      modalShown: {
+        ModalAddYourClub: false,
+        ModalLearnMore: false
+      }
+    };
+  },
+  closeModal: function(targetModal) {
+    var modalShownState = this.state.modalShown;
+    modalShownState[targetModal] = false;
+    this.setState(modalShownState);
+  },
+  showModal: function(e) {
+    var targetModal = e.target.getAttribute("data-modal-name");
+    var modalShownState = this.state.modalShown;
+    modalShownState[targetModal] = true;
+    this.setState(modalShownState);
+  },
   render: function() {
-    var theCtaLink = "fixme"; // CTA link should be the same for hero CTA and page bottom CTA
     return (
       <div>
         <HeroUnit image="/img/hero-clubs.jpg">
           <h1>Mozilla Learning Clubs</h1>
-          <div><Link to={theCtaLink} className="btn btn-awsm">Add Your Club</Link></div>
-          <p className="learn-more">or <Link to="fixme">find out more</Link> about us</p>
+          <div><a className="btn btn-awsm" onClick={this.showModal} data-modal-name="ModalAddYourClub">Add Your Club</a></div>
+          <div><p className="learn-more">or <a onClick={this.showModal} data-modal-name="ModalLearnMore">find out more</a> about us</p></div>
```

Comment:
I wonder if you should make `showModal` take the modal name as its argument and use `showModal.bind`, similar to how you did with [`closeModal`](https://github.com/mozilla/teach.webmaker.org/pull/269/files#diff-043f02d87f17791d954cfbed0983455fR233)? It makes the style more consistent, and I think not relying on the DOM could also make it easier to unit test.